openest/generate/smart_curve.py

```diff
@@ -351,22 +356,30 @@ def __init__(self, curve, offset):
         self.offset = offset
 
     def __call__(self, ds):
-        return self.curve(ds) - self.offset
+        return self.curve(ds) + self.offset
 
-    def get_univariate(self):
-        return curve.ShiftedCurve(self.curve.get_univariate(), self.offset)
+    @property
+    def univariate(self):
+        return curve.ShiftedCurve(self.curve.univariate, self.offset)
     
     def format(self, lang):
-        return formatting.build_recursive({'latex': r"(%s - " + str(self.offset) + ")",
-                                           'julia': r"(%s - " + str(self.offset) + ")"},
+        return formatting.build_recursive({'latex': r"(%s + " + str(self.offset) + ")",
+                                           'julia': r"(%s + " + str(self.offset) + ")"},
                                           lang, self.curve)
 
 class ClippedCurve(curve.ClippedCurve, SmartCurve):
-    def get_univariate(self):
-        return curve.ClippedCurve(self.curve.get_univariate(), self.cliplow)
+    @property
+    def univariate(self):
+        return curve.ClippedCurve(self.curve.univariate, self.cliplow)
+
+class OtherClippedCurve(curve.OtherClippedCurve, SmartCurve):
+    @property
+    def univariate(self):
+        return curve.ClippedCurve(self.clipping_curve.univariate, self.curve.univariate, self.clipy)
```

Comment:
There is an extra arg to `curve.ClippedCurve`?

Reply:
Should have been `OtherClippedCurve`.